docs/source/en/model_doc/conditional_detr.mdx

```diff
@@ -0,0 +1,66 @@
+<!--Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# conditional_detr
+
+## Overview
+
+The conditional_detr model was proposed in [ConditionalDETR](https://arxiv.org/abs/2108.06152) by Depu Meng, Xiaokang Chen, Zejia Fan, Gang Zeng, Houqiang Li, Yuhui Yuan, Lei Sun, Jingdong Wang.
+
+The abstract from the paper is the following:
+
+*The recently-developed DETR approach applies the
+transformer encoder and decoder architecture to object detection and achieves promising performance. In this paper, we handle the critical issue, slow training convergence,
+and present a conditional cross-attention mechanism for
+fast DETR training. Our approach is motivated by that the
+cross-attention in DETR relies highly on the content embeddings for localizing the four extremities and predicting the
+box, which increases the need for high-quality content embeddings and thus the training difficulty.
+Our approach, named conditional DETR, learns a conditional spatial query from the decoder embedding for
+decoder multi-head cross-attention. The benefit is that
+through the conditional spatial query, each cross-attention
+head is able to attend to a band containing a distinct region, e.g., one object extremity or a region inside the object box. This narrows down the spatial range for localizing the distinct regions for object classification and box
+regression, thus relaxing the dependence on the content embeddings and easing the training. Empirical results show
+that conditional DETR converges 6.7× faster for the backbones R50 and R101 and 10× faster for stronger backbones
+DC5-R50 and DC5-R101. Code is available at https://github.com/Atten4Vis/ConditionalDETR.*
+
+
+This model was contributed by [DepuMeng](https://huggingface.co/DepuMeng).
+The original code can be found [here](https://github.com/Atten4Vis/ConditionalDETR).
+
+
+## ConditionalDETRConfig
+
+[[autodoc]] ConditionalDETRConfig
+
+## ConditionalDETRFeatureExtractor
+
+[[autodoc]] ConditionalDETRFeatureExtractor
+    - __call__
+    - pad_and_create_pixel_mask
+    - post_process
+    - post_process_segmentation
+    - post_process_panoptic
```

Comment:
There's probably no need for a new feature extractor, see comments below

Reply:
Check my reply to the other comment